analysis/__main__.py

```diff
@@ -106,6 +107,19 @@ def summarize_setting(summary, version, seconds, path, data):
             data = data.split(",")[0].split("(")[0].strip()
             data = data.replace("@TM@", "(TM)").replace("@R@", "(R)").replace("@C@", "(C)")
 
+            if "nvidia" in data.lower() or "geforce" in data.lower() or "quadro" in data.lower():
+                brand = "NVIDIA"
+            elif "intel" in data.lower():
+                brand = "Intel"
+            elif "amd " in data.lower() or "radeon" in data.lower():
+                brand = "AMD"
+            elif "apple" in data.lower():
+                brand = "Apple"
+            else:
+                brand = "Unknown"
```

Comment:
What happens when there's inevitably a large amount of cards that appear as unknown due to the summarising?

Reply:
huh? I don't understand what you mean here.

Reply:
Arbitrarily grouping everything that doesn't fit the above conditonals into "unknown" doesn't feel like a good solution

Reply:
And as always, such comments are not helping me at all. Why doesn't it feel good? What do you need for it to do feel good? Do you have suggestions? Work with me here :D